task2-bash/script.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+mkdir -p tmp/guessanumber
+cp ../task1-simple-program/*.c ./tmp/guessanumber/ 
+tar czf guessanumber.tar.gz ./tmp/guessanumber
+mkdir release
+cp guessanumber.tar.gz ./release/
+rm guessanumber.tar.gz
+cd ./tmp/guessanumber/
```

Comment:
Just a suggestion: 
Assuming that ./tmp/guessanumber/ is a local folder, the following lines can be replaced with
`rm -rf ./tmp`